kubernetes/schema_pod_spec.go

```diff
@@ -153,6 +153,41 @@ func podSpecFields(isUpdatable bool) map[string]*schema.Schema {
 			ValidateFunc: validateTerminationGracePeriodSeconds,
 			Description:  "Optional duration in seconds the pod needs to terminate gracefully. May be decreased in delete request. Value must be non-negative integer. The value zero indicates delete immediately. If this value is nil, the default grace period will be used instead. The grace period is the duration in seconds after the processes running in the pod are sent a termination signal and the time when the processes are forcibly halted with a kill signal. Set this value longer than the expected cleanup time for your process.",
 		},
+		"tolerations": {
```

Comment:
The attribute name should be singular, as in `toleration`.
This is because in HCL, to express the *tolerations* list with multiple elements, you would have to write multiple distinct blocks of:
```hcl
toleration {
  key = "key"
  operator = "Equal"
  value = "value"
  effect = "NoSchedule"
}

toleration {
  key = "key"
  operator = "Exists"
  effect = "NoSchedule"
}
```

Reply:
Ah yes of course, that's a good point